lib/ssh_scan/constants.rb

```diff
@@ -18,6 +21,6 @@ module Constants
                             "2d63747200000021686d61632d6d64352c686d61632d736861312c686d61632" +
                             "d726970656d6431363000000021686d61632d6d64352c686d61632d73686131" +
                             "2c686d61632d726970656d64313630000000046e6f6e65000000046e6f6e650" +
-                            "00000000000000000000000006e05b3b4").unhexify
+                            "00000000000000000000000006e05b3b4").unhexify)
```

Comment:
This isn't exactly what I was going for here, although this is a step in the right direction.

You could also achieve the same result with KeyExchangeInit because it's a bindata object class type....

default_key_init = SSHScan::KeyExchangeInit.read("...binary...data...")

What I was striving for on this was to build the object piece by piece attribute by attribute...

    default_key_init = SSHScan::KeyExchangeInit.new()
    default_key_init.macs = blah
    default_key_init.encryption = blah
    ....